solutions/devsprint-michael-douglas-2/DeliveryAppTests/RestaurantCellViewSnapshotTests.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  RestaurantCellViewSnapshotTests.swift
+//  DeliveryAppTests
+//
+//  Created by Pedro Francisco Tres on 22/02/22.
+//
+
+import XCTest
+import SnapshotTesting
+@testable import DeliveryApp
+
+class RestaurantCellViewSnapshotTests: XCTestCase {
+    
+    let size = CGSize(width: 300, height: 80)
+
+    func testRestaurantCellViewSnapshotTests() throws {
```

Comment:
Uma coisa que esqueci de comentar no outro PR, mas que irei passar com vocês amanhã, é em relação a boas práticas de nomenclatura de Testes Unitários. Seguindo as boas práticas, o correto seria o seu método se chamar:
```suggestion
    func test_restaurantCell_whenSnapshot_shouldSucceed() throws {
```

Reply:
Dúvida: o nome não precisa terminar em Tests ?

Reply:
Nops, por que o prefixo do método já é `func test_` 🚀

Reply:
Ajustado